packages/jest-matchers/src/matchers.js

```diff
@@ -37,20 +39,36 @@ const matchers: MatchersObject = {
       };
     }
   },
+
+  toBeTruthy(actual: any, expected: void) {
+    ensureNoExpected(expected);
+    const pass = !!actual;
+    let message;
+
+    pass
+      ? message = () => `expected '${stringify(actual)}' to be truthy`
+      : message = () => `expected '${stringify(actual)}' not to be truthy`;
+
+    return {message, pass};
+  },
+
+  toBeFalsy(actual: any, expected: void) {
+    ensureNoExpected(expected);
+    const pass = !actual;
+    let message;
+
+    pass
+      ? message = () => `expected '${stringify(actual)}' to be falsy`
+      : message = () => `expected '${stringify(actual)}' not to be falsy`;
+
+    return {message, pass};
+  },
 };
 
-// Remove circular references
-function stringify(obj: any): string {
-  const set = new Set();
-  return JSON.stringify(obj, (key, value) => {
-    if (typeof value === 'object' && value !== null) {
-      if (set.has(value)) {
-        return '[Circular]';
-      }
-      set.add(value);
-    }
-    return value;
-  });
+function ensureNoExpected(expected) {
+  if (typeof expected !== 'undefined') {
+    throw new Error('This matcher does not accept any arguments');
```

Comment:
This will be a lot better with the actual matcher name.

"Jest: toBeFalsy: This matcher does not accept any arguments."